.github/workflows/publish.yml

```diff
@@ -0,0 +1,51 @@
+name: publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Version to Checkout and Release'
+        required: true
+        default: 'v0.1.2'
+
+jobs:
+  publish:
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      -
+        uses: actions/checkout@master
+        with:
+          repository: rakyll/hey
+          ref: ${{ github.event.inputs.tag }}
+          fetch-depth: 1
+      -
+        name: Generate Binaries
+        run: |
+          CGO_ENABLED=0 GOOS=linux go build -mod=readonly -o bin/hey && \
+          CGO_ENABLED=0 GOOS=darwin go build -mod=readonly -o bin/hey-darwin && \
+          CGO_ENABLED=0 GOOS=windows go build -mod=readonly -o bin/hey-windows.exe && \
+          CGO_ENABLED=0 GOOS=linux GOARCH=arm GOARM=6 go build -mod=readonly -o bin/hey-armhf && \
+          CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -mod=readonly -o bin/hey-arm64
+      -
+        name: Create release
```

Comment:
I would rather create the release tag in the GitHub UI and have the action run automatically after that. Could you modify to look more like the arkade/k3sup versions?

Reply:
Sure!